api/deploy_test.go

```diff
@@ -30,10 +30,9 @@ func TestDeployService(t *testing.T) {
 		archive, err := xarchive.GzippedTar(path, nil)
 		require.NoError(t, err)
 
-		service, validationError, err := a.DeployService(archive, nil, DeployServiceStatusOption(statuses))
+		_, validationError, err := a.DeployService(archive, nil, DeployServiceStatusOption(statuses))
```

Comment:
why removing the check of the service in this tests ?

Reply:
Because it wasn't necessary here - it's not responsible of api to check what length has hash